src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -1608,6 +1610,7 @@ public sealed partial class MetadataReader
         public System.Reflection.Metadata.AssemblyDefinition GetAssemblyDefinition() { throw null; }
         public System.Reflection.Metadata.AssemblyFile GetAssemblyFile(System.Reflection.Metadata.AssemblyFileHandle handle) { throw null; }
         public System.Reflection.Metadata.AssemblyReference GetAssemblyReference(System.Reflection.Metadata.AssemblyReferenceHandle handle) { throw null; }
+        public System.Reflection.AssemblyName GetAssemblyName() { throw null; }
```

Comment:
Do we want to expose this API as public or make it instead internal? As the API proposal was to add this methods in AssemblyDefinition and AssemblyReference.